ci/includes/pinned-tools.sh

```diff
@@ -24,10 +24,15 @@ function getUnityDir() {
 }
 
 UNITY_DIR="$(getUnityDir)"
+export UNITY_HOME="${UNITY_DIR}"
 export UNITY_EXE="${UNITY_DIR}/Editor/Unity.exe"
 
 export LINTER="cleanupcode.exe"
 
 export MSBUILD="$(powershell  –ExecutionPolicy Bypass ./ci/find-msbuild.ps1)"
 
-export NUNIT3_CONSOLE="code_generator/packages/NUnit.ConsoleRunner.3.8.0/tools/nunit3-console.exe"
+export NUNIT3_CONSOLE="tools/DocsLinter/packages/NUnit.ConsoleRunner.3.8.0/tools/nunit3-console.exe"
+
+DOTNET_VERSION="$(dotnet --version)"
+
+export MSBuildSDKsPath="/c/Program Files/dotnet/sdk/${DOTNET_VERSION}/Sdks"
```

Comment:
Why is this required?

Reply:
because otherwise the TC agents can't find `Microsoft.Net.Sdk`. Will need to check with IT today as it seems to be related to not installing .NET Core via Visual Studio, but for now I left it in here until it's resolved

Reply:
Use `${PROGRAM_FILES}` as a hedge for now.